pom.xml

```diff
@@ -84,6 +84,7 @@
         <typetools.version>0.5.0</typetools.version>
         <cucumber-expressions.version>6.0.1</cucumber-expressions.version>
         <datatable.version>1.1.3</datatable.version>
+        <project.Automatic-Module-Name>io.cucumber.jvm</project.Automatic-Module-Name>
```

Comment:
The pom is not released as a jar. So I don't think it needs a module name.

Reply:
if the default changes to <project.Automatic-Module-Name></project.Automatic-Module-Name> then your other comments can simply have the property removed, from memory the property needs to be defined as maven-jar-plugin can't allow conditional options so for these jars add this extra configuration

Reply:
made default empty property so that koltin-java8 and continious-deployment also pick up that property